scripts/report-download-images.py

```
@@ -111,101 +126,117 @@ def main():
-        if age is not None and age > args.stale_days:
-            outdated.append((age, b, board_support.get(b, "?"), newest_ver, newest.strftime("%Y-%m-%d")))
-    outdated.sort(reverse=True)
-    out.append(f"## ⏳ Outdated boards — newest live image older than {args.stale_days} days ({len(outdated)})")
+    for b, (k, v, d) in dl_newest.items():
+        if k[:2] < current_line:  # behind the current major.minor line
+            age = (now - d).days if d else None
+            outdated.append((current_line[0]*100 + current_line[1] - (k[0]*100 + k[1]),  # minors behind
+                             b, board_support.get(b, "?"), v, d.strftime("%Y-%m-%d") if d else "?", age))
+    outdated.sort(key=lambda t: (-t[0], t[3]))
+    out.append(f"## ⏳ Outdated boards on the download — behind the current {current_str} line ({len(outdated)})")
+    out.append(f"_Newest per-board image on `dl.armbian.com` is older than the current release line._")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)